TalkAloud/TalkAloud/ViewControllers/AudioRecordingsViewController.swift

```diff
@@ -29,20 +30,19 @@ class AudioRecordingsViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         audioRecordings = AudioManager.sharedInstance.loadAudioRecordings(with: nil)!
+        tagModelDataSource = getAllTags()
         recordingsTableView.reloadData()
     }
     
     //==================================================
-    // MARK: - Actions
+    // MARK: - Private Methods
     //==================================================
-    
-    @IBAction func tappedLeftButton(_ sender: Any) {
-        presentTagTableViewController()
+    private func getAllTags() -> [TagModel] {
```

Comment:
Why return smth from this function?

Reply:
I was thinking that I needed to update the dataSource whenever I called this but after reading your comment I know I didn't need to do that.